tests/sample-envoy-config-dump-bookbuyer-no-rds-dynamic-route-virtual-host-domains.json

```diff
@@ -0,0 +1,1368 @@
gBBf2oiADYCACAADQBBvB8hAwNAIAMoAgAiAEEIaiEDIAANAAtBlBxBfzYCAAsLKQEBfwJAQYQCEIsBIgBFDQAgAEF8ai0AAEEDcUUNACAAQYQCEBQLIAALC+wQAwBBhAgL7wksBAAACQAAAAoAAAALAAAADAAAAA0AAAAOAAAADwAAABAAAAARAAAArAsAAHIGAADsBgAATAwAAEAEAAAzJF8wAAAAAAAAAABgBQAAEgAAABMAAAAUAAAAFQAAABYAAAAXAAAAGAAAABkAAAAaAAAAGwAAABwAAAAdAAAAHgAAAB8AAAAgAAAAIQAAACIAAAAjAAAAJAAAACUAAAAmAAAAJwAAACgAAABvc21fcmVxdWVzdF90b3RhbAByZXNwb25zZV9jb2RlAHNvdXJjZV9uYW1lc3BhY2UAc291cmNlX2tpbmQAc291cmNlX25hbWUAc291cmNlX3BvZABkZXN0aW5hdGlvbl9uYW1lc3BhY2UAZGVzdGluYXRpb25fa2luZABkZXN0aW5hdGlvbl9uYW1lAGRlc3RpbmF0aW9uX3BvZABvc21fcmVxdWVzdF9kdXJhdGlvbl9tcwCsCwAANAYAAEQGAABsaXN0ZW5lcl9kaXJlY3Rpb24AcHJveHlfZ2V0X2N1cnJlbnRfdGltZV9uYW5vc2Vjb25kcygmdCkAbWV0cmljIGZpZWxkcy5zaXplKCkgIT0gdGFncy5zaXplKCkAZGVmaW5lTWV0cmljKHR5cGUsIG4sICZtZXRyaWNfaWQpADpzdGF0dXMAb3NtLXN0YXRzLW5hbWVzcGFjZQBvc20tc3RhdHMta2luZABvc20tc3RhdHMtbmFtZQBvc20tc3RhdHMtcG9kADEyU3RhdHNDb250ZXh0AACsCwAAUAYAAFwGAAA3Q29udGV4dAAAAABMDAAAZAYAADExQ29udGV4dEJhc2UATlN0M19fMjEwX19mdW5jdGlvbjZfX2Z1bmNJMyRfME5TXzlhbGxvY2F0b3JJUzJfRUVGTlNfMTB1bmlxdWVfcHRySTdDb250ZXh0TlNfMTRkZWZhdWx0X2RlbGV0ZUlTNl9FRUVFalAxMVJvb3RDb250ZXh0RUVFAABMDAAA9AYAAE5TdDNfXzIxMF9fZnVuY3Rpb242X19iYXNlSUZOU18xMHVuaXF1ZV9wdHJJN0NvbnRleHROU18xNGRlZmF1bHRfZGVsZXRlSVMzX0VFRUVqUDExUm9vdENvbnRleHRFRUUAAAAAAAAARAYAACkAAAAqAAAAFAAAABUAAAArAAAAFwAAABgAAAAZAAAAGgAAABsAAAAcAAAAHQAAAB4AAAAfAAAALAAAACEAAAAiAAAAIwAAAC0AAAAlAAAAJgAAACcAAAAuAAAAcGx1Z2luX3Jvb3RfaWQAcHJveHlfZ2V0X3Byb3BlcnR5KCJwbHVnaW5fcm9vdF9pZCIsIHNpemVvZigicGx1Z2luX3Jvb3RfaWQiKSAtIDEsICZyb290X2lkX3B0ciwgJnJvb3RfaWRfc2l6ZSkAAAAAAACECAAALwAAADAAAAAxAAAAMgAAACsAAAAzAAAAGAAAABkAAAAaAAAANAAAADUAAAA2AAAANwAAADgAAAA5AAAAOgAAADsAAAA8AAAAPQAAAD4AAACsCwAAkAgAAFwGAAAxMVJvb3RDb250ZXh0AHN0ZDo6YmFkX2Z1bmN0aW9uX2NhbGwAAAAAAAAAAMwIAAAFAAAAPwAAAEAAAACsCwAA2AgAAPwKAABOU3QzX18yMTdiYWRfZnVuY3Rpb25fY2FsbEUAQYQSC+UGAgAAAAMAAAAFAAAABwAAAAsAAAANAAAAEQAAABMAAAAXAAAAHQAAAB8AAAAlAAAAKQAAACsAAAAvAAAANQAAADsAAAA9AAAAQwAAAEcAAABJAAAATwAAAFMAAABZAAAAYQAAAGUAAABnAAAAawAAAG0AAABxAAAAfwAAAIMAAACJAAAAiwAAAJUAAACXAAAAnQAAAKMAAACnAAAArQAAALMAAAC1AAAAvwAAAMEAAADFAAAAxwAAANMAAAABAAAACwAAAA0AAAARAAAAEwAAABcAAAAdAAAAHwAAACUAAAApAAAAKwAAAC8AAAA1AAAAOwAAAD0AAABDAAAARwAAAEkAAABPAAAAUwAAAFkAAABhAAAAZQAAAGcAAABrAAAAbQAAAHEAAAB5AAAAfwAAAIMAAACJAAAAiwAAAI8AAACVAAAAlwAAAJ0AAACjAAAApwAAAKkAAACtAAAAswAAALUAAAC7AAAAvwAAAMEAAADFAAAAxwAAANEAAABhbGxvY2F0b3I8VD46OmFsbG9jYXRlKHNpemVfdCBuKSAnbicgZXhjZWVkcyBtYXhpbXVtIHN1cHBvcnRlZCBzaXplAGJhc2ljX3N0cmluZwB2ZWN0b3IAc3RkOjpleGNlcHRpb24AAAAAAAD8CgAAQQAAAEIAAABDAAAATAwAAAQLAABTdDlleGNlcHRpb24AAAAAAAAAACgLAAACAAAARAAAAEUAAACsCwAANAsAAPwKAABTdDExbG9naWNfZXJyb3IAAAAAAFgLAAACAAAARgAAAEUAAACsCwAAZAsAACgLAABTdDEybGVuZ3RoX2Vycm9yAFN0OXR5cGVfaW5mbwAAAEwMAAB1CwAArAsAACEMAACECwAArAsAAMwLAACMCwAAAAAAAPALAABHAAAASAAAAEkAAABKAAAASwAAAEwAAABNAAAATgAAAE4xMF9fY3h4YWJpdjExN19fY2xhc3NfdHlwZV9pbmZvRQAAAKwLAAD8CwAAmAsAAE4xMF9fY3h4YWJpdjEyMF9fc2lfY2xhc3NfdHlwZV9pbmZvRQBOMTBfX2N4eGFiaXYxMTZfX3NoaW1fdHlwZV9pbmZvRQAAAAAAAACYCwAARwAAAE8AAABJAAAASgAAAEsAAABQAAAAUQAAAFIAQewYCwPwD1A="
```

Comment:
I'd trim this and all other `inline_bytes` to an empty string.  Two reasons:
- these are certs - a security scan will raise a false positive (false because these are test certs)
- too much text - hard to scroll through - so I'd remove them for readability